neptunecontrib/api/__init__.py

```diff
@@ -25,23 +25,23 @@
     get_property_columns,
     get_system_columns,
     strip_prefices,
-    pickle_and_log_artifact,
```

Comment:
If we remove it here, then the deprecation notices are a bit difficult to access :)

Reply:
fixed